doc/website/blog/ogent.md

```diff
@@ -0,0 +1,559 @@
+---
+title: Announcing "ogent" - An Extension to Automatically Generate a Server Implementation for "entoas" generated OpenAPI Specification Documents 
```

Comment:
Precise, but long ! Let's come up with something more catchy. Needs to storm through social media!

Keep ten accuracy for an opening sentence or sub headline

Reply:
Also intro desperately needs a diagram